common/pb/moderator_commands.proto

```diff
@@ -17,3 +18,10 @@ message Command_BanFromServer {
     optional string visible_reason = 5;
     optional string clientid = 6;
 }
+
+message Command_GetBanHistory {
+    extend ModeratorCommand {
+        optional Command_GetBanHistory ext = 1001;
+    }
+    optional string user_name = 1;
```

Comment:
I had never thought about it since the only method for calling to get some one's ban history is from the context menu and they being logged in so the field shouldn't ever end up blank.  But that's not to say some were else the command may be used in the future called else were.  I think my vote is to leave it as is and work the future behavior in when we get there. What was your acronym again?  Something about not writing code for the future or trying to predict the future.... i dont recall. :)